rsts/user/getting_started/create_first.rst

```diff
@@ -14,18 +14,15 @@ You can save some effort by cloning the ``flytesnacks`` repo, and re-initializin
   git init
   cd python
 
-now open the "Makefile" and change the first line to ``IMAGE_NAME=myflyteproject``
 
-Let's also remove the existing python task so we can write one from scratch. ::
-
-  rm single_step/edges.py
+Let's take a look at how easy it is to use Flyte by writing a task from scratch.
 
 Creating a Project
 ******************
 
 In Flyte, workflows are organized into namespaces called "Projects". When you register a workflow, it must be registered under a project.
 
-Lets create a new project called ``myflyteproject``. Use the project creation endpoint to create the new project ::
+For example, to create a new project called ``myflyteproject``: use the project creation endpoint to create the new project ::
```

Comment:
create a project should be flyte-cli, can we get rid of the curl command?

Reply:
sure, done @kumare3